llvm/lib/MC/RepoObjectWriter.cpp

```diff
@@ -561,13 +561,17 @@ RepoObjectWriter::toPstoreLinkage(GlobalValue::LinkageTypes L) {
   case GlobalValue::ExternalLinkage:
     return pstore::repo::linkage_type::external;
   case GlobalValue::LinkOnceAnyLinkage:
+    return pstore::repo::linkage_type::link_once_any;
   case GlobalValue::LinkOnceODRLinkage:
+    return pstore::repo::linkage_type::link_once_odr;
   // TODO: WeakAnyLinkage implements as linkonce for now to allow LLVM to build
   // correctly. Implement support for weak linkage in the future.
```

Comment:
I think so. As discussed in issue #10, "We possibly need a new issue to discuss how best to implement weak in the repo".

Reply:
I think that this change rather supersedes that discussion since, as of this change, we’re exposing the full set of LLVM linkage types in the repository.

Reply:
Ok, the comment has been removed.